Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -1,6 +1,16 @@
 module Hardware
   class CPU
     class << self
+      OPTIMIZATION_FLAGS_LINUX = {
```

Comment:
I'd like to remove `:penryn` from `OPTIMIZATION_FLAGS`. I'd prefer that it use `-march=native` like the others rather than `-march=core2 -msse4.1`.